src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -52,7 +52,7 @@ public function __toString()
         foreach ($this->headers as $name => $values) {
             $name = implode('-', array_map('ucfirst', explode('-', $name)));
             foreach ($values as $value) {
-                $content .= sprintf("%-{$max}s %s\r\n", $name.':', $value);
+                $content .= sprintf("%-${max}s %s\r\n", $name.':', $value);
```

Comment:
personally, I prefer `{$max}`; it generates much cleaner tokens when parsing (think e.g. static analyzers)